From 93ef90b0a92e19ae9b5f6f828cf02a5e790db59b Mon Sep 17 00:00:00 2001
From: Henry Wu <Henry_Wu@quantatw.com>
Date: Thu, 19 Mar 2026 16:31:12 +0800
Subject: [PATCH 3/5] configuration: santabarbara: configure MMC as
 MCTPI2CTarget in BiosPost

Mark MMC devices as MCTPI2CTarget and set PowerState to BiosPost,
indicating they are valid after BIOS POST.

The link of the gerrit:
https://gerrit.openbmc.org/c/openbmc/entity-manager/+/88430

Signed-off-by: Henry Wu <Henry_Wu@quantatw.com>
---
 .../meta/santabarbara/santabarbara_rainbow.json           | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/configurations/meta/santabarbara/santabarbara_rainbow.json b/configurations/meta/santabarbara/santabarbara_rainbow.json
index 26e8318..ad5476a 100644
--- a/configurations/meta/santabarbara/santabarbara_rainbow.json
+++ b/configurations/meta/santabarbara/santabarbara_rainbow.json
@@ -15,6 +15,14 @@
             },
             "Name": "Santabarbara_Rainbow_CPLD_$bus / 2 + 2 * 3 % 5 * 3 + 1 / 4",
             "Type": "LatticeLCMXO3D_9400Firmware"
+        },
+        {
+            "Address": "0x21",
+            "Bus": "$bus / 2 + 2 * 3 % 5 * 3 + 1 / 4 + 102",
+            "Inventory": "Rainbow",
+            "Name": "MMC",
+            "PowerState": "BiosPost",
+            "Type": "MCTPI2CTarget"
         }
     ],
     "Name": "Santa Barbara Rainbow module $bus / 2 + 2 * 3 % 5 * 3 + 1 / 4",
-- 
2.52.0

